packages/docusaurus-theme-classic/codeTranslations/de.json

```
@@ -42,13 +42,13 @@
   "theme.docs.sidebar.expandButtonTitle": "Seitenleiste ausklappen",
   "theme.docs.sidebar.responsiveCloseButtonLabel": "Menü schließen",
   "theme.docs.sidebar.responsiveOpenButtonLabel": "Menü öffenen",
-  "theme.docs.versions.latestVersionLinkLabel": "latest version",
-  "theme.docs.versions.latestVersionSuggestionLabel": "For up-to-date documentation, see the {latestVersionLink} ({versionLabel}).",
-  "theme.docs.versions.unmaintainedVersionLabel": "This is documentation for {siteTitle} {versionLabel}, which is no longer actively maintained.",
-  "theme.docs.versions.unreleasedVersionLabel": "This is unreleased documentation for {siteTitle} {versionLabel} version.",
-  "theme.lastUpdated.atDate": " on {date}",
-  "theme.lastUpdated.byUser": " by {user}",
-  "theme.lastUpdated.lastUpdatedAtBy": "Last updated{atDate}{byUser}",
+  "theme.docs.versions.latestVersionLinkLabel": "letzte Version",
+  "theme.docs.versions.latestVersionSuggestionLabel": "Für die aktuellste Dokumentation bitte auf {latestVersionLink} ({versionLabel}) gehen.",
+  "theme.docs.versions.unmaintainedVersionLabel": "Das ist die Dokumentation für {siteTitle} {versionLabel} und wird nicht weiter gewartet.",
+  "theme.docs.versions.unreleasedVersionLabel": "Das ist die unveröffentlichte Dokumentation für {siteTitle} {versionLabel}.",
+  "theme.lastUpdated.atDate": " am {date}",
+  "theme.lastUpdated.byUser": " von {user}",
+  "theme.lastUpdated.lastUpdatedAtBy": "Letztes Update {atDate}{byUser}",
```

```suggestion
  "theme.lastUpdated.lastUpdatedAtBy": "Letztes Update{atDate}{byUser}",
```

the space is omitted on purpose as it's already inside the `atDate` string